packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -191,6 +191,7 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
       setCurrentPage(newState.page);
       setCurrentCallState(newState.call?.state);
     };
+    updateCallWithChatPage(callWithChatAdapter.getState());
```

Comment:
This is a fix for a bug I discovered by the new hermetic test -- we depended on _some_ state change to correctly show the `ChatComposite` and `PeoplePane` that are part of the `CallWithChatComposite`. It is probably not noticed in practice because state changes occur frequently.